test/missing_semicolon.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title></title>
+    <script>
+        var xmlHttp = new XMLHttpRequest();
+        xmlHttp.open( "GET", '../src/js.cookie.js', false );
+        xmlHttp.send( null );
+        window.scriptContent = '(function (){ return {}; })() ';
+        window.scriptContent += xmlHttp.responseText;
+        window.scriptContent += ';window.ok = true;';
```

Comment:
Is there really a reason for changing `window.ok = true` after it is loaded? Can't we just check if the test throws an `Error`?